frontends/systolic-lang/gen-systolic.py

```diff
@@ -858,7 +858,7 @@ def counter():
     # build the while loop with condition cond_reg.
     # num repeats = (top_length - 1) + (left_length - 1) + (top_depth - 1) + 5 + 1
     cond_reg_port = comp.get_cell("cond_reg").port("out")
-    while_loop = cb.while_(cond_reg_port, None, while_body)
+    while_loop = cb.while_(cond_reg_port, while_body)
```

Comment:
The one case where a `while` was being constructed without a `comb` group!

Reply:
Makes sense! This is where we want to build a loop that executes every cycle using the magic of `static` bodies in `while` loops.